udata_hydra/config.py

```diff
@@ -22,7 +22,7 @@
 # crawl url if last check is older than
 SINCE = '1w'
 # seconds to wait for between batches
-SLEEP_BETWEEN_BATCHES = 60
+SLEEP_BETWEEN_BATCHES = 10
```

Comment:
Woops, I pushed my config by mistake
```suggestion
SLEEP_BETWEEN_BATCHES = 60
```